gatelevel_readers/reconstruct_verilog/tests/RAM/RAM18Kx2_MEM_CHECK/smallSizeTest/run.sh

```diff
@@ -0,0 +1,9 @@
+mkdir build
```

Comment:
This run.sh is not a standard entry point, it should be called from a Makefile.
When stepping in this directory, typing "make" or "make test" should call this script. The entire Raptor source tree uses "make" as entry points in each subsystem. Let's not deviate from that practice.

Once the Makefile is created, then invoke it (make or make test) from the Github Actions, so the test is actually run automatically when checking in code.

Reply:
I will.